commands/deploy.go

```diff
@@ -194,7 +195,10 @@ func IsValidNodeApp(dir string) (errs []error) {
 
 	return errs
 }
-
+// Split helps differentiate between different directory delimiters. / or \
+func Split(r rune) bool {
```

Comment:
Why not [filepath.Split()](https://golang.org/pkg/path/filepath/#Split)?
>for manipulating filename paths in a way compatible with the target operating system-defined file paths.

Reply:
TIL that exists. I just used google.

Reply:
It doesn't split the dir into an array.